### PR TITLE
Improve travis

### PR DESCRIPTION
**Pull request by dbatyai**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,109 @@
...
   - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
   - sudo add-apt-repository -y ppa:andykimpe/cmake
   - sudo apt-get -qq update
-  - sudo apt-get install -y freeglut3-dev libegl1-mesa-dev libgles2-mesa-dev
   - sudo apt-get -qq install g++-4.8 gcc-4.8
   - sudo apt-get -qq install cmake
   - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.8 90
   - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.8 90
 
+
```
Comment (szledan) [inline .travis.yml]: Do we need this extra line?

**Comment by kkristof** [comment]
lgtm

File: tools/scripts/unittest.py [hunk 2]
```diff
@@ -38,10 +38,15 @@ def run_unittest(args):
...
+    ret = build.configure(build.get_args())
 
-    return subprocess.call([path.join(build_path, 'bin', 'unit')])
+    if not ret:
+      ret = build.build_unit(args)
+
+    if not ret:
+      ret = subprocess.call([path.join(build_path, 'bin', 'unit')])
+
+    return ret
```
Comment (szledan) [inline tools/scripts/unittest.py]: Extra line(?)

**Comment by szledan** [comment]
Another nice work.. Thanks.

lgtm

**Pull request merged**